integrations/milvus-document-store.md

```diff
@@ -149,7 +185,8 @@ rag_pipeline = Pipeline()
 rag_pipeline.add_component("text_embedder", OpenAITextEmbedder())
 rag_pipeline.add_component("retriever", MilvusEmbeddingRetriever(document_store=document_store, top_k=3))
 rag_pipeline.add_component("prompt_builder", PromptBuilder(template=prompt_template))
-rag_pipeline.add_component("generator", OpenAIGenerator(generation_kwargs={"temperature": 0}))
+rag_pipeline.add_component("generator", OpenAIGenerator(api_key=Secret.from_token(os.getenv("OPENAI_API_KEY")),
```

Comment:
Not a must, but we want to encourage using the `OpenAIChatGenerator`. If you want to use that component, you need to use ChatPromptBuilder instead of PromptBuilder too.